src/backend/api/Fusion.Resources.Logic/Workflows/InternalRequestDirectWorkflowV1.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Fusion.Resources.Database.Entities;
+using System.Collections.Generic;
+
+namespace Fusion.Resources.Logic.Workflows
+{
+
+    public class InternalRequestDirectWorkflowV1 : WorkflowDefinition
+    {
+        public const string CREATED = "created";
+        public const string APPROVAL = "approval";
+        public const string PROVISIONING = "provisioning";
+
+        public override string Version => "v1";
+        public override string Name => "Personnel request of direct type";
+
+        public InternalRequestDirectWorkflowV1()
```

Comment:
There are no approval step for the direct workflow. It just goes directly to provisioning.. ?